src/main/java/com/teragrep/cfe_39/consumers/kafka/MockKafkaConsumerFactoryTemp.java

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
Mock and Temp inside main sources?

Reply:
Renamed to MockKafkaConsumerFactory in commit 57c6805d036db4ea9c7fca8aa81a5bfbbdd5d2b6
Should the class be moved to test sources instead? It's located in main sources because the factory was implemented that way in pth_06 too.